packages/core/src/analytics/posthogAiBetaServerClient/featureFlagEvaluation.ts

```diff
@@ -0,0 +1,57 @@
+import { aiLogger } from "@oakai/logger";
+import { kv } from "@vercel/kv";
+import type { PostHog } from "posthog-node";
+
+const KV_KEY = "posthog-feature-flag-local-evaluation";
+const ONE_DAY = 24 * 60 * 60 * 1000;
+const ONE_MINUTE = 60 * 1000;
+
+const log = aiLogger("analytics:feature-flags");
+
+const setKv = async (response: Response) => {
+  const value = await response.text();
+  await kv.set(KV_KEY, value, { ex: ONE_MINUTE });
```

Comment:
Does this need `kv.set<some type>`?

Reply:
The thing is, I don't know if it matters to us what the shape is if we're just caching it. Posthog's own code that calls fetch doesn't actually put a type on it either, so it could be a bit of extra effort for something that we don't need to validate and could drift from the type

https://github.com/PostHog/posthog-js-lite/blob/b853833c126ba713eb25696c96a3f0a6193ea2f0/posthog-node/src/feature-flags.ts#L422